sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -3763,14 +3763,14 @@
       "labelWidth": "",
       "labelMargin": "",
       "tag": "p",
-      "className": "text-muted text-center",
+      "className": "text-muted text-full-justify",
       "attrs": [
         {
           "attr": "",
           "value": ""
         }
       ],
-      "content": "Collection and Use of Information. The information included in this form is collected under ss. 26(c) and 26(e) of the Freedom of Information and Protection of Privacy Act, R.S.B.C. 1996, c. 165. Upon pressing the “Submit” button you are confirming that you have reviewed this statement. The information you provide will be used in confirming your identity. If you have any questions about the collection and use of this information, contact the Executive Director, StudentAid BC, Ministry of Post Secondary and Future Skills, PO Box 9173, Stn Prov Govt, Victoria BC, V8W 9H7, telephone 1-800-561-1818 (toll-free in Canada/U.S.) or +1-778-309-4621 from outside North America.",
+      "content": "Collection and Use of Information. The information included in this form is collected under ss. 26(c) and 26(e) of the Freedom of Information and Protection of Privacy Act, R.S.B.C. 1996, c. 165. Upon pressing the “Submit” button you are confirming that you have reviewed this statement. The information you provide will be used in confirming your identity. If you have any questions about the collection and use of this information, contact the Executive Director, StudentAid BC, Ministry of Post-Secondary Education and Future Skills, PO Box 9173, Stn Prov Govt, Victoria BC, V8W 9H7, telephone 1-800-561-1818 (toll-free in Canada/U.S.) or +1-778-309-4621 from outside North America.",
```

Comment:
Unless I am missing something, this was not captured in ticket but it is good that you fixed the text.